finance-tracker/app/assets/stylesheets/custom.css.scss

```diff
@@ -16,3 +16,8 @@
   display: inline-block;
   margin-bottom: 1rem;
 }
+
+.user-name {
+  margin-top: 3.5%;
+  padding-right: 10px;
+}
```

Comment:
What does this have to do with adding column to user table?

Reply:
edited the PR title